net-libs/cppzmq/cppzmq-4.6.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="High-level CPP Binding for ZeroMQ"
+HOMEPAGE="https://github.com/zeromq/cppzmq"
+SRC_URI="https://github.com/zeromq/${PN}/archive/v${PV}.tar.gz ->  ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+
+# Except for test these are used to alter installed cmake files:
+IUSE="test drafts static-libs"
+RESTRICT="!test? ( test )"
+
+DEPEND=">=net-libs/zeromq-4.3.1[drafts?,static-libs?]
+	test? ( dev-cpp/catch )"
+RDEPEND=">=net-libs/zeromq-4.3.1[drafts?,static-libs?]"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-disable-static.patch"
+	"${FILESDIR}/${PN}-use-system-catch2.patch"
+)
+
+src_configure() {
+	local mycmakeargs=(
+		-DCPPZMQ_CMAKECONFIG_INSTALL_DIR="/usr/$(get_libdir)/cmake/"
+		-DBUILD_STATIC_LIBS="$(usex static-libs)"
```

Comment:
Do you need static libs for anything? If not, remove the flag altogether.

Reply:
sure, I do. BTW library is header only, the flag only influences the way installed cmake modules detect zeromq, what compiler definitions are going to be exported there and how the tests will be built.

Reply:
But… how does that affect the end user? If it's header only, you should really just use shared libs at build time and don't care for runtime.

Reply:
I'm working on a project which has a requirement to be statically linked. It affects end user if the end user is a software developer who has projects with these kinds of requirements.

Reply:
ok, I've managed to get rid of the flag, tests will be run against dynamic libraries,I guess